make sure easybuild flags and libs are used for llvmlite
March 7th 2018 by Sam Moors (Vrije Universiteit Brussel - VUB)
--- llvmlite-0.22.0/ffi/Makefile.linux.orig	2018-02-15 23:41:57.000000000 +0100
+++ llvmlite-0.22.0/ffi/Makefile.linux	2018-03-07 11:45:06.159770575 +0100
@@ -5,9 +5,9 @@
 CXX_FLTO_FLAGS ?= -flto
 LD_FLTO_FLAGS ?= -flto -Wl,--exclude-libs=ALL
 
-CXXFLAGS = $(LLVM_CXXFLAGS) $(CXX_FLTO_FLAGS)
-LDFLAGS = $(LLVM_LDFLAGS) $(LD_FLTO_FLAGS)
-LIBS = $(LLVM_LIBS)
+CXXFLAGS := $(CXXFLAGS) $(LLVM_CXXFLAGS) $(CXX_FLTO_FLAGS)
+LDFLAGS := $(LDFLAGS) $(LLVM_LDFLAGS) $(LD_FLTO_FLAGS)
+LIBS := $(LIBS) $(LLVM_LIBS)
 INCLUDE = core.h
 SRC = assembly.cpp bitcode.cpp core.cpp initfini.cpp module.cpp value.cpp \
 	  executionengine.cpp transforms.cpp passmanagers.cpp targets.cpp dylib.cpp \
